mol_dqn/chemgraph/dqn/run_dqn_test.py

```diff
@@ -24,20 +24,20 @@
 
 from absl import flags
 from absl.testing import flagsaver
+from tensorflow_core import core
+
 import tensorflow as tf
-from tensorflow import gfile
-from mol_dqn.chemgraph.mcts import deep_q_networks
-from mol_dqn.chemgraph.mcts import run_dqn
-from mol_dqn.chemgraph.tensorflow import core
+import deep_q_networks
+import run_dqn
 
 
 class RunDQNTest(tf.test.TestCase):
 
   def setUp(self):
     super(RunDQNTest, self).setUp()
-    self.mount_point = tempfile.mkdtemp(dir=flags.FLAGS.test_tmpdir)
```

Comment:
I think you can use `from tensorflow import flags`: https://www.tensorflow.org/api_docs/python/tf/app/flags (or use `absl.flags` like you do elsewhere).

Reply:
The problem is `test_tmpdir` is not set within the `flags` module. It gives an error of 

```
Traceback (most recent call last):
  File "run_dqn_test.py", line 39, in setUp
    self.mount_point = tempfile.mkdtemp(dir=flags.FLAGS.test_tmpdir)
  File "/Users/odin/miniconda3/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 85, in __getattr__
    return wrapped.__getattr__(name)
  File "/Users/odin/miniconda3/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 472, in __getattr__
    raise AttributeError(name)
AttributeError: test_tmpdir
```